_docs/updating-files-managed-with-modulesync.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Updating Files Managed With ModuleSync
+date: 2021-06-30
+summary: How to proceed to update files managed by ModuleSync in Vox Pupuli templates.
+---
+
+## What is ModuleSync?
+
+[ModuleSync] is used at Vox Pupuli to syncronise common files across the various modules we manage.  It helps us to update all our modules a consistent way.
+
+## How do I update a file managed by ModuleSync?
+
+The configuration shared between our modules is setup in the [modulesync_config] repository.
+
+At the root of this repository, is a directory named `moduleroot` which contains all the templates used to generate the files ModuleSync manages.  Fill-in a PR against the [modulesync_config] repository to make changes to files managed by ModuleSync.
+
+If your changes do not apply to *all* modules managed by Vox Pupuli, use `@config['your_custom_key']` to tune the templates expansion.  The default value of `your_custom_key` can be set in ` config_defaults.yml` in the [modulesync_config] repository, and the custom value for a specific module in a `.sync.yml` file at the root of that module repository.
```

Comment:
what? where? :) An example would be helpful since this requires changing things in two different repos.

Reply:
Added an example.  I really feel unable to write comprehensible English, rewording in proper words would be appreciated!